tests/stability/test_array.py

```diff
@@ -5,11 +5,19 @@
 
 from ..utils_test import cluster_memory, scaled_array_shape, wait
 
+try:
+    import scipy  # noqa: F401
+
+    has_scipy = True
+except ImportError:
+    has_scipy = False
```

Comment:
Didn't use importorskip because, since there's only one test in this module, the cluster would be started needlessly

Reply:
❤️ I would suggest you add this as an explicit comment in code, otherwise someone (like me) may have an autopilot moment and replace it with `importorskip` without thinking.